.github/workflows/main.yml

```diff
@@ -0,0 +1,12 @@
+name: Greetings
+
+on: [issues]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/first-interaction@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          issue-message: " Hi👋 there! Welcome to the project💟\n Thank you and congrats🎉for opening your very first issue in this project. AnitaB.org is an inclusice community,committed to \n creating a safe and positive environment.🌸please adhere to [Code of Conduct🏽🙌](https://github.com/anitab-org/open-source-programs-backend/blob/develop/CODE_OF_CONDUCT.md) \n You may submit a PR on the same if you like! If you want to repost a bug🐞 please follow our [REPORTING_GUIDELINES.md](https://github.com/anitab-org/open-source-programs-backend/blob/develop/REPORTING_GUIDELINES.md) template. Also, please make sure you include steps to reproduce it and be patient we will get back to you as soon as possible👀. \n \n Also feel free to join us on [AnitaB.org OpenSource Zulip Community.](https://anitab-org.zulipchat.com/)🤝🏻 We have different streams for each active repo's for discussions.👨🏻‍💻👩🏻‍💻 Hope you have a great time here :-) "
```

Comment:
Why only new issue comment is there, we need to add PR welcome comments and merge comments too.
I think the issue clearly states what we need  for the welcome bot🤔

Reply:
I will make changes EXACTLY according to <a href="https://github.com/anitab-org/bridge-in-tech-web/pull/94/files">this</a> PR. Will make the changes today positively. If i work according to this file, i guess the build will also pass. Thus template is good to follow.

Reply:
Ya sure, and about the build, it is going to fail on every Pull request on the backend (means here) it's not a contributors fault though, some API key-related problem is there because of which every build here on GitHub at the backend is failing

Reply:
okay. I have made the changes you asked.